### ISSUE TITLE
Download case model from Gryphon

### ISSUE THREAD

**Issue by marhew** [bptlab/chimera#60]
As developer and case worker I want to download deployed case models from Chimera to deploy it to another instance. The case model overview page should therefore offer a button to download the deployed case  model. 

- [ ] Extend the REST-API of Chimera with an endpoint to download case models
- [ ] Construct the Json/XML representation for case models (reverse parsing) **or** store the case model file when a case model is deployed from Gryphon for later use
- [ ] Send this Json/XML representation to the frontend **or** make the stored case model files accessible via the web server

**Comment by janwestphal306** [comment]
Concerning the second point. It is also possible to store the json content of the casemodel inside the database regarding that we are already stroring the xml representations of the fragments which needed most of the place in the json content?

**Comment by marhew** [comment]
Yeah, the case model could also be stored in the database. However, storing it as a file on the server would probably we simpler and allow straightforward access via the web browser (no need to read it from the DB when it is requested). In the end, these are implementation details, that you will have to figure out.
